cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -139,6 +140,23 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
     richText.toolbar.embedDropdown.should('be.disabled');
   });
 
+  it('invoke onAction callback when rendering links', () => {
+    cy.setInitialValue(documentWithLinks);
+
+    // Necessary for reading the correct LocalStorage values as we do
+    // the initial page load on the beforeEach hook
+    cy.reload();
+    cy.editorActions().should(
```

Comment:
Ideally, we would assert that this only fires once the links were actually fetched as the purpose is to measure how long that takes. So if these were to be fired early, the test should fail. Would be great if it's not too complex, otherwise let's focus on other fixes.